### PR TITLE
Rabble logo v1 ep

### PR DESCRIPTION
**Pull request by epachols**
> made intro logo, put in home page component (should eventually put it back up in app and fix re-render of whole app based on game state chage so it doesn't revamp when you clear game history..)
> added hook to display:none for logo container, no visible difference but it's better to remove invisible elements from the page flow entirely, to avoid weird possible future touch interactions.

### CODE REVIEW

File: src/webapp/features/home/index.tsx [hunk 1]
```diff
@@ -18,14 +19,17 @@ const Home = () => {
   }
 
   return (
+    <React.Fragment>
```
Comment (apachols) [inline src/webapp/features/home/index.tsx]: Ooh here's something fun - here's an even better syntax for React.fragment:
```
    <>
      <RabbleLogo />
      ...
      </div>
    </>
```

Let's change it to use this less verbose syntax!!  🏄

File: src/webapp/features/rabble_logo/rabbleLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+      clearTimeout(disappear);
+    }
+  }, [])
+
+    return (
+      <>
+    <div className={styles.rabbleLogo__container} id={"disappear"}>
+      <div className={styles.flex}>
+        <div className={styles.cube}>
+          <div className={`${styles.wall} ${styles.front}`}>
```
Comment (apachols) [inline src/webapp/features/rabble_logo/rabbleLogo.tsx]: Ooh i bet we can eliminate some of the repetition from this component by making a <Cube> component and passing it the side-letters as props!

File: src/webapp/features/rabble_logo/rabbleLogo.module.css [hunk 3]
```diff
@@ -0,0 +1,183 @@
+@media only screen and (max-width: 550px) {
```
Comment (apachols) [inline src/webapp/features/rabble_logo/rabbleLogo.module.css:1]: Tested on ios, the alignment is a tiny bit off - a bit to the right, and cut off by that dumb bar at the bottom:

ios-rabble

For a little to the right, i'm sure there's a CSS fix.

For the bar at the bottom, let's consider making the cubes a bit smaller on mobile?

Also, this points at our need to get you a way to test on IOS... I have an idea, we can discuss on our next call.

File: src/webapp/features/rabble_logo/rabbleLogo.tsx [hunk 4]
```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { useEffect } from "react";
+import styles from './rabbleLogo.module.css';
+
+const RabbleLogo = () => {
+  // this just display:none's the container after a countdown to diminish the chance of weird interactions with it hiding in the z-index.
+  useEffect(() => {
+    const disappear = setTimeout(() => {
+      let logoContainer = document.getElementById("disappear");
```
Comment (apachols) [inline src/webapp/features/rabble_logo/rabbleLogo.tsx]: This timer is working pretty well, but something tells me there's a more react-y way to do this, with useRef.  I'll think on it a bit; probably isn't necessary to change it now.

Comment (epachols) [reply src/webapp/features/rabble_logo/rabbleLogo.tsx]: ooh, okay! got it - I was at first just making sure I could do it, happy to improve next round!

File: src/webapp/features/rabble_logo/rabbleLogo.module.css (hunk 3 above)
Comment (epachols) [reply src/webapp/features/rabble_logo/rabbleLogo.module.css:1]: interesting that chrome doesn't include the utility bar to check screen size. Noted!

Comment (epachols) [reply src/webapp/features/rabble_logo/rabbleLogo.module.css:1]: should be super easy once I can preview in xcode as all of those measurements are based in VH and %, no probs to make it smaller and more central

**Comment by apachols** [comment]
LGTM 👍 whoa this PR is beautiful now!  good suggestion on splitting up the components.  Merging now, thanks!! 🚀

**Pull request merged**